packages/core/src/components/graphs/treemap.ts

```diff
@@ -336,7 +346,12 @@ export class Treemap extends Component {
 							"graph_element_mouseout_fill_update"
 						)
 					)
-					.style("fill", null);
+					.style("fill", (d: any) => {
+						const fillColor = self.model.getFillColor(
+							d.parent.data.name
+						);
+						return fillColor ? fillColor : null;
```

Comment:
wouldn't `getFillColor` return null anyways if it doesn't like the input provided?

Reply:
it returns `undefined` if no provide color

Reply:
it should return null...?

Reply:
it should return null